### ISSUE TITLE
about sizeof(AuthorizationStatusTexts) 

### ISSUE THREAD

**Issue by onlyone-wuqingzhong** [mliras/O_C_P_P_Client#2]
in 
manageSendLocalListRequest
{
    for (int i=0; i<sizeof(AuthorizationStatusTexts); i++)
    {
	if (strcmp(AuthorizationStatusTexts[i],status_str)) status=i;
   }
}

I think it would be better to change the way it is.
 for (int i=0; i<sizeof(AuthorizationStatusTexts)/sizeof(AuthorizationStatusTexts[0]); i++)

**Comment by mliras** [comment]
Yes, you are right. I was a very newbie in C at the moment of writing this code.

Thanks!
